### DISCUSSION TITLE
display specific events in same calendar

### DISCUSSION THREAD

**Discussion by alemao8** [pretix/pretix#3128] [Q&A]
Hello.

Is it possible to display in the same calendar some of the total events ?
Let's say I have 10 events of one organizer and I want to display in the same calendar 3 events of total 10.

Thank you in advance

**Comment by wiffbi** [comment]
If you use the widget, that should be possible. Although AFAIK you cannot pass IDs of events/subevents to display in the widget, but you can use the filter-attribute to filter by meta-attributes defined on the event. See the last parapgraph here: https://docs.pretix.eu/en/latest/user/events/widget.html#multi-event-selection

**Comment by alemao8** [comment]
I managed to filter specific events in the same calendar by doing this.
I used the organizer's configuration (Organizer->Settings->Event metadat) to create the metadata property and then on each event I set a value (Event settings->General->Basics->meta data field) in order to filter the list of events I want to use in the same calendar.
This worked fine
